src/connector/src/source/nexmark/source/combined_event.rs

```diff
@@ -66,6 +69,62 @@ pub fn new_combined_event(event: Event) -> CombinedEvent {
     }
 }
 
+pub fn get_event_data_types_with_names(
+    event_type: Option<EventType>,
+    row_id_index: Option<usize>,
+) -> Vec<(String, DataType)> {
+    let mut fields = match event_type {
+        None => {
+            vec![
+                ("event_type".to_owned(), DataType::Int64),
+                (
+                    "person".to_owned(),
+                    DataType::Struct(get_person_struct_type().into()),
+                ),
+                (
+                    "auction".to_owned(),
+                    DataType::Struct(get_auction_struct_type().into()),
+                ),
+                (
+                    "bid".to_owned(),
+                    DataType::Struct(get_bid_struct_type().into()),
+                ),
+            ]
+        }
+        Some(EventType::Person) => {
+            let struct_type = get_person_struct_type();
+            struct_type
+                .field_names
+                .into_iter()
+                .zip_eq_fast(struct_type.fields)
+                .collect_vec()
+        }
+        Some(EventType::Auction) => {
+            let struct_type = get_auction_struct_type();
+            struct_type
+                .field_names
+                .into_iter()
+                .zip_eq_fast(struct_type.fields)
+                .collect_vec()
+        }
+        Some(EventType::Bid) => {
+            let struct_type = get_bid_struct_type();
+            struct_type
+                .field_names
+                .into_iter()
+                .zip_eq_fast(struct_type.fields)
+                .collect_vec()
+        }
+    };
+
+    if let Some(row_id_index) = row_id_index {
+        // _row_id
+        fields.insert(row_id_index, ("_row_id".to_owned(), DataType::Serial));
```

Comment:
May use `row_id_column_desc` instead of hard-coding here.

Reply:
fix by using `row_id_column_name`